src/main/webapp/WEB-INF/themes/desktop/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
Please add a note if this is a libsass-only file and I don't have to care about it.